lib/phoenix_live_view/html_algebra.ex

```diff
@@ -308,7 +308,7 @@ defmodule Phoenix.LiveView.HTMLAlgebra do
 
   # Final clause: multiple lines
   defp text_to_algebra([], _, acc),
-    do: acc |> Enum.reverse() |> tl() |> concat() |> force_unfit()
+    do: acc |> Enum.reverse() |> tl() |> concat()
```

Comment:
Not sure why we had this `force_unit` in the first place. But removing that fixes the issue 😅